<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_downloading</class>
 <widget class="QWidget" name="Page_downloading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <column>
    <property name="text">
     <string>选择</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>大小</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>进度</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>下载速度</string>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>剩余时间</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bt_delete">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>69</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="text">
    <string>正在下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_add">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_start_one">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_pause_one">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>20</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_set_downloads">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置下载目录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
